### ISSUE TITLE
Support disable x-pack security

### ISSUE THREAD

**Issue by meggarr** [spujadas/elkx-docker#10]
Hi,

Would you please add env variable so as to disable the security for the purpose of dev usage? I believe it will be much convenient.

Thanks!

**Comment by spujadas** [comment]
Good idea! I quickly hacked a development mode (set `DEVELOPMENT_MODE` to `1` to check it out in `sebp/elk:latest`; build in progress at the time of writing), let me know how it works out for you.
